addons/mail/models/mail_channel.py

```diff
@@ -779,6 +777,7 @@ def channel_info(self):
                     info['custom_channel_name'] = partner_channel.custom_channel_name
                     info['is_pinned'] = partner_channel.is_pinned
                     info['last_interest_dt'] = partner_channel.last_interest_dt.strftime(DEFAULT_SERVER_DATETIME_FORMAT)
+                    info['message_unread_counter'] = partner_channel.message_unread_counter
```

Comment:
`partner_channel` should be `with_prefetch` for all others `member_of_current_user_by_channel` at once